civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -45,16 +43,19 @@
          */
         function modalDocument (id, role, mode) {
           DocumentService.get({ id: id })
-            .then(function(data) {
-              if (!data) {
-                throw new Error('Requested Document is not available');
-              }
-              vm.openModalDocument(data[0], role, mode);
-              vm.cacheContacts(data).then(function () {
-                $rootScope.isLoading = false;
-              });
+            .then(function (data) {
```

Comment:
where did the "throw new Error" go?

Reply:
It's not required not as the exception (if no data is found) will be handled by`.catch(function (reason) {` block.